containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -62,6 +62,29 @@
 
 namespace Kokkos {
 
+namespace Impl {
+
+template <typename ViewType, typename... P, typename Property, typename... Args>
+auto allocate_with_property_if_compatible(
+    const Impl::ViewCtorProp<P...> &alloc_prop,
+    [[maybe_unused]] const Property &property, Args &&...args) {
+  using alloc_prop_t = Impl::remove_cvref_t<decltype(alloc_prop)>;
```

Comment:
I think

```suggestion
  using alloc_prop_t = Impl::ViewCtorProp<P...>;
```

Is slightly more readable

Reply:
was kept for consistency with the rest of the file